js/script.js

```diff
@@ -0,0 +1,40 @@
+let bookLib = [];
+const book = {};
+function getBook() {
+  book.title = document.querySelector('.title').value;
+  book.author = document.querySelector('.author').value;
+  return book;
+}
+
+function addBook(books) {
+  const bookLi = document.querySelector('#book-list');
+  const book = document.createElement('LI');
+  book.setAttribute('style', 'id:list');
+  book.innerHTML = `<h3> ${books.title} </h3> <p>${books.author} </p>`;
+  book.style.listStyle = 'none';
+  const delBook = document.createElement('button');
+  delBook.innerHTML = 'Delete';
+  delBook.addEventListener('click', () => {
+    book.innerHTML = '';
+  });
+  book.appendChild(delBook);
+  bookLi.appendChild(book);
+}
+const addBtn = document.querySelector('.addBook');
+addBtn.addEventListener('click', () => {
+  const book = getBook();
+  bookLib.push(book);
+  localStorage.setItem('bookLib', JSON.stringify(bookLib));
+  addBook(book);
+});
+
```

Comment:
- [ ] Good trying to fix the changes required by the previous reviewer. But as he mentioned to make sure that Data is preserved in the browser's memory by using [localStorage](https://github.com/microverseinc/curriculum-html-css/blob/main/local_storage.md). Right now, books from local storage can be displayed to your UI but if a user delete a book it doesn't update data from local storage. **Make sure you fix this as it is one of major requirements of the project.**